website/server/docgenHelpers.js

```diff
@@ -1,5 +1,12 @@
 "use strict";
-var b = require('react-docgen/node_modules/recast').types.builders;
+var b;
+try {
+  // npm 3 probably
+  b = require('recast').types.builders;
```

Comment:
No. Node's algo doesn't descend to look for modules, only up.

```
.
├── node_modules
│   ├── react-docgen
│   │   └── node_modules
│   │       └── recast
│   └── something-else
└── server
    └── docgenHelpers.js
```

docgenHelpers will first look for `server/node_modules` (nope), then `$PWD/node_modules` (yay) then look for `recast` in that directory. It won't go looking recursively down from there there. Then it goes and continues walking up the filesystem (I think? I forget how this part works).